core/common/src/thrift/common.thrift

```diff
@@ -28,6 +28,11 @@ enum CommandType {
   Persist = 5,  // Ask the worker to persist a file for lineage
 }
 
+enum TTtlAction {
```

Comment:
I think enum makes sense for configurations like Ttl actions, where the permitted values are limited (0/1). Even enums are encoded as i32.

Note: I have still not changed the thrift/proto definitions, as part of this commit. We can discuss, if we agree on non-enum, then I'll change